cli_reference/tkn_cli/installing-tkn.adoc

```diff
@@ -0,0 +1,18 @@
+[id='installing-tkn']
+= Installing tkn
+include::modules/common-attributes.adoc[]
+include::modules/openshift-pipelines-attributes.adoc[]
+:context: installing-tkn
+
+toc::[]
+
+Use the `tkn` CLI to manage {op-title} from a terminal. The following section describes how to install `tkn` on different platforms.
```

Comment:
{op-title} Just wanted to check if this attribute is as per the latest changes made to the attributes file?